news/2021-09-16-osu-world-cup-2021-registrations-now-open.md

```diff
@@ -26,11 +26,11 @@ For more details, refer to the **[osu! World Cup 2021 wiki article](/wiki/Tourna
 
 ## Prizes
 
-The osu! World Cup 2021 offers a $5,000 prize pool, along with limited edition goods!
+The osu! World Cup 2021 offers a $5,000 prize pool, along with limited edition goods! Prize amounts for each placement have been slightly adjusted from 2020 for a more fair distribution.
 
-- **1st:** 50% of the prize pool, exclusive limited-edition merch, profile badge, "osu! Champion" user title for one year
-- **2nd:** 34% of the prize pool, exclusive limited-edition merch, profile badge
-- **3rd:** 16% of the prize pool, exclusive limited-edition merch, profile badge
+- **1st:** 48% of the prize pool, exclusive limited-edition merch, profile badge, "osu! Champion" user title for one year
```

Comment:
Just for documentation purposes, is this objectively fairer? Is it based off other tournaments or some discussion that came up?

Reply:
The main drive behind this change was so that 3rd place finalists get a value that is more "acceptable" at a negligible cost, seeing as the only difference between the second and third place finishers is one match:

1st place goes from $312.50 for each player to $300
2nd from $225 each to $200
3rd from $87.50 each to $125

Reply:
There are a few examples of other recent tournaments doing this as well, [corsace open](https://open.corsace.io/info) for instance (while the percentages are not exactly the same as ours, the overall idea stays the same: make the distinction between 2nd and 3rd place finalists less extreme)

Reply:
Any reason to not make it 50/30/20 in that case? I was more curious about the first place reduction.

Reply:
one could argue the proposal is punishing first place too, i dunno what that logic is. but i guess as long as the community can understand this even if i can't?

will leave this PR open for a day or two in case anyone else wants to chime in